src/Apps/Sell/Routes/AdditionalRoutes/ConditionRoute.tsx

```diff
@@ -1,33 +1,48 @@
+import { ConditionRoute_submission$key } from "__generated__/ConditionRoute_submission.graphql"
+import { ArtworkConditionEnumType } from "__generated__/useUpdateMyCollectionArtworkMutation.graphql"
 import { DevDebug } from "Apps/Sell/Components/DevDebug"
+import { Formik } from "formik"
+import { graphql, useFragment } from "react-relay"
+import {
+  Box,
+  Clickable,
+  Flex,
+  Join,
+  Select,
+  Spacer,
+  Text,
+  TextArea,
+} from "@artsy/palette"
 import { SubmissionLayout } from "Apps/Sell/Components/SubmissionLayout"
 import { SubmissionStepTitle } from "Apps/Sell/Components/SubmissionStepTitle"
 import { useSellFlowContext } from "Apps/Sell/SellFlowContext"
-import { ConditionRoute_submission$key } from "__generated__/ConditionRoute_submission.graphql"
-import { Formik } from "formik"
 import * as React from "react"
-import { graphql, useFragment } from "react-relay"
 import * as Yup from "yup"
+import { useMemo, useState } from "react"
+import { acceptableConditionsForSubmission } from "Apps/Sell/Utils/acceptableConditionsForSubmission"
+import { ConditionInfoModal } from "Apps/Artwork/Components/ArtworkDetails/ConditionInfoModal"
 
 const FRAGMENT = graphql`
   fragment ConditionRoute_submission on ConsignmentSubmission {
-    title
     myCollectionArtwork {
+      artworkId: internalID
       condition {
+        description
```

Comment:
I think the description value should be coming from `conditionDescription.details` instead https://github.com/artsy/eigen/blob/6db51a8103ac59d50983e437f2cb538b6362ab9d/data/schema.graphql#L2368
It seems to be the one used in prod and from cms https://github.com/artsy/eigen/blob/6db51a8103ac59d50983e437f2cb538b6362ab9d/data/schema.graphql#L2368

Reply:
You're right, will address